proto/buf/alpha/registry/v1alpha1/doc.proto

```diff
@@ -377,6 +387,40 @@ message FieldOptions {
   int32 jstype = 4;
 }
 
+// Option provides information for a descriptor option that can be show in documentations.
+message Option {
+  // The name of the option.
```

Comment:
```suggestion
  // The name of the option. Custom options will be in the form:
  //     [fully.qualified.extension.Name]
```

Reply:
An alternative to this convention, mainly the use of leading `[` and `]` in the string, is to have another field here that indicates whether this is a custom option or not.
```proto
  bool is_custom = 2; // would need to renumber value to 3
```

Reply:
Great idea, I included `is_custom` to the message